### ISSUE TITLE
nthFibonacciNumber[javascript]

### ISSUE THREAD

**Issue by Vijay-K-2003** [iiitv/algos#781]
#### This issue is about [nthFibonacciNumber](https://en.wikipedia.org/wiki/Fibonacci_number).

- [x] Issue Title is in form `Algo/DS Name [Language]`.
- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This Algo/DS is worth adding to this repository.
- [x] Assign this issue to me (I want to work on this).
	- [x] I ensure that I am not already assigned to 2 or more issues.
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

Please assign me this under contribute-a-thon.

![image](https://user-images.githubusercontent.com/75903092/135673981-16159a6a-f16b-4f28-8758-3c075aff29ee.png)

**Comment by vidit2512** [comment]
@Vijay-K-2003 GFI 👍.

**Comment by Vijay-K-2003** [comment]
@vidit2512 Kindly assign me BellmanFord also.

**Comment by Vijay-K-2003** [comment]
Issue #723

**Comment by vidit2512** [comment]
> @vidit2512 Kindly assign me BellmanFord also.

@Vijay-K-2003 Why are you mixing up these 2 issues here, if you want to talk about any other algorithm kindly discuss it in another issue and as you have told earlier, you have worked on 2 algorithms in a single PR in the master branch, this will not work.
Kindly make separate branches to work on separate issues simultaneously and then make PR.
